app/src/main/java/com/instructure/template/projectCodeHere/layouts/MainActivity.java

```diff
@@ -123,7 +124,13 @@ public void onResponse(@NotNull Call call, @NotNull Response response) {
                      Log.i(TAG, "Response body: " + response.body().toString());
                      Log.i(TAG, courses.toString());
                      for (GetCourses.CoursesResponse c : courses) {
-                         db.addDrawerItems(new PrimaryDrawerItem().withName(c.getName()));
+                         double tmp = 0;
+                         for (GetEnrollments.EnrollmentResponse i : enrollments) {
+                             if (i.getCourse_id()==c.getId()) {
+                                 tmp = i.getGrades().getCurrent_score();
+                             }
+                         }
+                         db.addDrawerItems(new PrimaryDrawerItem().withName(c.getName() + "\t\t%" + tmp));
```

Comment:
Swap percentages and the number around.
`db.addDrawerItems(new PrimaryDrawerItem().withName(c.getName() + "\t\t" + tmp + "%"));`